lib/avro_turf/confluent_schema_registry.rb

```diff
@@ -122,7 +124,7 @@ def post(path, **options)
 
   def request(path, **options)
     options = { expects: 200 }.merge!(options)
-    response = @connection.request(path: path, **options)
+    response = @connection.request(path: "#{@path_prefix}#{path}", **options)
```

Comment:
```suggestion
    path = File.join(@path_prefix, path) unless @path_prefix.nil?
    response = @connection.request(path: path, **options)
```

I don't think we should expect users of this API to know that they need to have the prefix end with `/`. I think the above version should work with or without trailing `/`.